app/src/main/res/layout/player_custom_layout_tv.xml

```diff
@@ -250,25 +250,28 @@
                         android:id="@+id/player_video_title_rez"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        android:gravity="end"
+                        android:gravity="center"
                         android:textColor="@color/white"
                         android:textSize="16sp"
                         tools:text="1920x1080" />
 
-                    <FrameLayout
-                        android:id="@+id/player_episode_filler_holder"
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:layout_gravity="end"
-                        android:layout_marginTop="2dp">
 
-                        <com.google.android.material.button.MaterialButton
-                            android:id="@+id/player_episode_filler"
-                            style="@style/SmallBlackButton"
-                            android:text="@string/filler" />
-                    </FrameLayout>
                 </LinearLayout>
 
+                <FrameLayout
```

Comment:
.... I am not sure you want to move this. This might crash as player_custom_layout_tv and player_custom_layout MUST have the same xml. Try running this test https://github.com/recloudstream/cloudstream/blob/d3828eeafed0fd4fbeb32c4d37dee2126296b564/app/src/androidTest/java/com/lagradost/cloudstream3/ExampleInstrumentedTest.kt#L98

Reply:
![image](https://github.com/recloudstream/cloudstream/assets/17493070/fb459681-1fa0-4077-a2e0-3c3097fc4e25)

from what I have encountered before, as long as the element exists with same Id there will be no issue, you have to be cautious when removing it or changing the id for it.